runtime/doc/options.txt

```diff
@@ -4821,9 +4824,11 @@ A jump table for the options with a short description can be found at |Q_op|.
 		45%	relative position in the file
 	If 'rulerformat' is set, it will determine the contents of the ruler.
 	Each window has its own ruler.  If a window has a status line, the
-	ruler is shown there.  Otherwise it is shown in the last line of the
-	screen.  If the statusline is given by 'statusline' (i.e. not empty),
-	this option takes precedence over 'ruler' and 'rulerformat'
+	ruler is shown there.  If a window doesn't have a status line and
+	'cmdheight' is 0, the ruler is not shown.  Otherwise it is shown in
```

Comment:
```suggestion
	'cmdheight' is 0, the ruler is not shown. Otherwise it is shown in
```

Reply:
Using two spaces seems more consistent here.

Reply:
Oh? Ignore this then